hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/client/clustering/run/strategy/SparkConsistentBucketClusteringExecutionStrategy.java

```diff
@@ -79,15 +94,19 @@ public HoodieData<WriteStatus> performClusteringWithRecordsRDD(HoodieData<Hoodie
     HoodieWriteConfig newConfig = HoodieWriteConfig.newBuilder().withProps(props).build();
 
     RDDConsistentBucketBulkInsertPartitioner<T> partitioner = new RDDConsistentBucketBulkInsertPartitioner<>(getHoodieTable(), strategyParams, preserveHoodieMetadata);
+    addHashingChildNodes(partitioner, extraMetadata);
```

Comment:
I am a little confused by the name between ConsistentHashingBucketInsertPartitioner#addHashingChildrenNodes

Reply:
Here I made a new interface call `ConsistentHashingBucketInsertPartitioner ` to share the common method `addHashingChildrenNodes ` between `RDDConsistentBucketBulkInsertPartitioner ` and `ConsistentBucketIndexBulkInsertPartitionerWithRows`.  I didn't put the method `addHashingChildrenNodes` in their closest parent class `BulkInsertPartitioner` because It's all about consistent hashing bucket index and I don't want to affect other partitioners. What do you think about this design here?  Should I keep the current design, move this to the parent class, or duplicate the code for both partitioners?